bower.json

```diff
@@ -15,12 +15,16 @@
     "akc-meta": "git://github.com/akc42/akc-meta.git#b0a6bde0e",
     "file-upload": "~1.6.1",
     "material-progress": "^0.1.2",
-    "paper-tabs": "PolymerElements/paper-tabs#^1.6.2"
+    "paper-tabs": "PolymerElements/paper-tabs#^1.6.2",
+    "paper-input": "timvdlippe/paper-input#spinner-mixin"
```

Comment:
we should probably make an issue to put this back to the upstream version after merging